### ISSUE TITLE
Allow Disabling Prefixes in PrefixPrinters.

### ISSUE THREAD

**Issue by jenslauterbach** [pterm/pterm#170]
I am currently trying to use pterm and would like to offer my users an option to output as plain text.

I'd like to use prefix printers (`pterm.Info.Println("")`) throughout my code, but if I want to offer plain text output, the prefix becomes a problem.

It would be great to have a way to disable prefixes completely using a "global" switch, similar to `pterm.DisableColor()` or `pterm.DisableDebugMessages()`.

When the switch is used, the following behaviour would be great:

```
pterm.DisablePrefixes()
pterm.Info.Println("Hello World") // Output: Hello World

pterm.EnablePrefixes()
pterm.Info.Println("Hello World") // Output: [ INFO ] Hello World
```

**Comment by MarvinJWendt** [comment]
Hi @jenslauterbach, thanks for the feedback. Since the feature is very `PrefixPrinter` specific, I don't think that a global setting would fit here. But, I think a `pterm.EnablePlainTextOutput()` / `pterm.DisbleStyling()` method would make sense! This would fit your use case (probably even better) and I totally see the relevance to disable all styling (for raw text files, etc.). Thanks again for the suggestion :)

I will work on this in the next days 👍

**Comment by jenslauterbach** [comment]
Thank you very much @MarvinJWendt. Much appreciated.

**Comment by MarvinJWendt** [comment]
Hi @jenslauterbach, sorry for the delay. I am in my exam phase so I don't have much time to work on this. We might ship this feature at the end of the month.

**Comment by jenslauterbach** [comment]
@MarvinJWendt Thank you for the update. No need to explain yourself why it is taking longer. I wish you the best of luck for your exams.

**Comment by MarvinJWendt** [comment]
Hi @jenslauterbach, I am currently implementing this.

This is the current functionallity:

### Default behaviour
![image](https://user-images.githubusercontent.com/31022056/118151838-5e566d00-b414-11eb-9e47-10dd37951507.png)

### `pterm.DisableStyling()` called before
![image](https://user-images.githubusercontent.com/31022056/118151914-70d0a680-b414-11eb-83e3-8ff5643c865e.png)

As you can see, the prefixes won't be removed completely, so that users can still get the context of a message in `RawOutput` mode, and better distinguish between Errors, Warnings and Infos in log files.

If you want to remove the prefixes completely, you could do this:
```go
pterm.DisableStyling()
pterm.Debug.Prefix.Text = ""
pterm.Info.Prefix.Text = ""
pterm.Success.Prefix.Text = ""
pterm.Warning.Prefix.Text = ""
pterm.Error.Prefix.Text = ""
pterm.Fatal.Prefix.Text = ""
```
![image](https://user-images.githubusercontent.com/31022056/118152628-36b3d480-b415-11eb-96f5-04d0bb2eff2d.png)

This would remove every prefix. The problem with that is, that it can't be toggled back without creating a backup of every `PrefixPrinter` beforehand.

Does this still fit your need?

**Comment by MarvinJWendt** [comment]
So this is now implemented in [`v0.12.15`](https://github.com/pterm/pterm/releases/tag/v0.12.15), let me know if this fits your need and if you find anything else 😄

Styles can be disabled with `pterm.DisableStyling()`

**Comment by jenslauterbach** [comment]
@MarvinJWendt Looks like it will do what I wanted. I will try it as soon as I have some time. Thank you for implementing this new feature.
